assets/src/scss/new-identity/style.scss

```diff
@@ -121,18 +121,41 @@ table.spreadsheet-table.is-color-gp-green {
   }
 }
 
-a:where(.standalone-link) {
+a:where(.standalone-link, .split-two-column-item-link) {
+  --box-shadow--offset-inset: inset;
+  --box-shadow--offset-x: 0;
+  --box-shadow--offset-y: calc(-22px);
+
   width: fit-content;
   border-bottom: 2px solid var(--gp-green-500);
   font-weight: 700 !important;
   transition: 0.3s cubic-bezier(0.5, 0, 0, 1);
 
   &:hover {
-    box-shadow: inset 0 calc(-22px) var(--gp-green-100);
+    box-shadow:
+      var(--box-shadow--offset-inset)
+      var(--box-shadow--offset-x)
+      var(--box-shadow--offset-y)
+      var(--gp-green-100);
     text-decoration: none !important;
   }
 }
 
+.split-two-column-item-link {
+  --link--color: white;
+  --link--hover--color: white;
+
+  text-shadow: none !important;
```

Comment:
Maybe we could remove this right away, I mean without waiting for the new identity? We would need to double-check with Houssam of course, but I see that we also have a `text-shadow` on some other elements of that block (the title, subtitle, etc) and I think it looks a bit weird 😅

Reply:
I agree, but I think it should be in another PR that change

Reply:
I agree 👍 Can you please create a follow-up ticket?

Reply:
Here is [PLANET-7138](https://jira.greenpeace.org/browse/PLANET-7138) !